pages/data-providers/[data-provider-id]/compliance.jsx

```diff
@@ -40,26 +40,31 @@ const SidebarContent = ({ context: { oai, originalId, authors, title } }) => {
  *       of Layout.Main and explicitly pass props.
  *       We should get rid out of it!
  */
-const DepositDates = ({
-  className,
-  store,
-  tag: Tag = 'main',
-  ...restProps
-}) => (
+const Compliance = ({ className, store, tag: Tag = 'main', ...restProps }) => (
   <Tag className={[styles.container, className].join(' ')} {...restProps}>
-    <h1>Deposit compliance</h1>
+    <h1>REF 2021 Compliance</h1>
 
-    <Card className={styles.complianceLevel} tag="section">
+    <Card className={styles.dataOverview} tag="section">
       <h2>{texts.compliance.title}</h2>
-      {store.depositDates.complianceLevel ? (
-        <Markdown>
-          {texts.compliance.body.render({
-            amount: (100 - store.depositDates.complianceLevel).toFixed(1),
-          })}
-        </Markdown>
-      ) : (
-        <p>{texts.noData.body}</p>
-      )}
+      <div className={styles.numbers}>
+        <NumericValue
+          tag="div"
```

Comment:
`p` is fine here, a [paragraph](https://html.spec.whatwg.org/#paragraph) despite it's commonly a few sentences of text in the web it is a small chunk of self-contained information and address, fieldset or a number + its description can be wrapped into `p`.

Reply:
I didn't want to remove all padding and margin stuff from paragraph hence `div`